### PR TITLE
fix the edge issue

### PR DESCRIPTION
**Pull request by Karol-Chen**

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ router
     //user = await getUserByUserName(username);
     try {
       user = await getUserByUserName(username);
-      if (user)
+      if (!Object.keys(user).length === 0){}
```
Comment (Sharatt-Seenivasan) [inline routes/users.js]: What's going on here? Why is there nothing in the body of the if statement?

Comment (Karol-Chen) [reply routes/users.js]: It must be something wrong. There is an error report in the if body. You can check the route file at line 102.

File: routes/index.js [hunk 2]
```diff
@@ -11,8 +11,8 @@ const constructorMethod = (app) => {
     app.use('/', async (req, res) => {
         if(req.session.user){
             //console.log(await getUserByUserName(req.session.user.username))
-            const user = await getUserByUserName(req.session.user.username)
-            const icon = user.icon
+            // const user = await getUserByUserName(req.session.user.username)
+            const icon = req.session.user.icon
```
Comment (Sharatt-Seenivasan) [inline routes/index.js:15]: I don't believe that `req.session.user` has the property `icon`.

Comment (Karol-Chen) [reply routes/index.js:15]: actually if you have a try, you can find that it works well

**Review by Sharatt-Seenivasan — APPROVED** [review]

**Pull request merged**